### ISSUE TITLE
##manager.masthead.title## not displayed properly

### ISSUE THREAD

**Issue by brueyd** [pkp/pkp-lib#3225]
When setting the properties of a new journal in the Settings Wizard, the heading on the masthead section appears as follows
##manager.masthead.title##
![image](https://user-images.githubusercontent.com/34877697/34370978-cd31d462-ea7d-11e7-9d33-a81e68c200e3.png)

**Comment by asmecher** [comment]
What version of OJS are you using?

**Comment by brueyd** [comment]
Hi Alec,

I just installed the latest package from the downloads page: OJS 3.1.0-1

I think I noticed a similar error in the setup wizard in a couple of other places, but accidentally closed the wizard before completing it. Is there a way to reinvoke the setup wizard for an existing journal?

Thanks!
Doug


On Dec 28, 2017 09:45, Alec Smecher <notifications@github.com> wrote:

What version of OJS are you using?
